@here/harp-mapview/lib/poi/PoiRenderer.ts

```diff
@@ -603,7 +603,7 @@ export class PoiRenderer {
         minS = xOffset / imageWidth;
         maxS = (xOffset + width) / imageWidth;
 
-        const flipY = true;
+        const flipY = false;
         if (flipY) {
```

Comment:
... but why not removing first branch as whole ?

Reply:
I guess it can be removed. Maybe a good idea expose `flipY` as a parameter if we do need it later on?